### DISCUSSION TITLE
How about submit flareact to offical dashboard ?

### DISCUSSION THREAD

**Discussion by iamspark1e** [flareact/flareact#157] [Ideas]
I recently noticed the offical team made a page for packages ["Works on Workers"](https://workers.cloudflare.com/works) which are

> The best packages and libraries supported by Cloudflare Workers, chosen by our community.

Maybe we can submit flareact to community and let more developers know this convenient framework.

**Comment by tj-kev** [comment]
Good idea. I've submitted the request.
